lib/gcov/README.md

```diff
@@ -0,0 +1,77 @@
+# Unikraft Library for Profiling and Test Coverage
+
+This library introduces initial setup with config options for console, binary file, and memory output. It takes advantage of gcc-13's newest feature of Profiling and Test Coverage in Freestanding Environments.
+
+## Prerequisites
+
+One way to build with `gcc-13` is to use a `Docker` container - [DockerImage](https://hub.docker.com/_/gcc/)
```

Comment:
Although you mentioned it already on the previous section looking at this paragraph is a bit sudden and without context to someone who looks directly at this section. Please add a sentence before that this requires a minimum version of GCC-13.